Pipfile

```diff
@@ -8,6 +8,7 @@ django = "*"
 
 [dev-packages]
 flake8 = "*"
+pillow = "*"
```

Comment:
Can you explain this line?

Reply:
This is a plugin @ShirleyFichman is using in order to save images to the database.